supervised/preprocessing/eda.py

```diff
@@ -150,7 +150,7 @@ def compute(X, y, eda_path):
                 fout.write("# Exploratory Data Analysis\n")
                 fout.write("\n\n[<< Go back](../README.md)\n")
 
-                for i, row in df.iterrows():
+                for row in df.to_dict(orient="records"):
```

Comment:
Have you done benchmarks on how much this speeds up iteration? Is there a memory tradeoff?

I also use df.iterrows(), but I'm glad to see a nicer alternative!